packages/http-signature-utils/src/utils/signatures.ts

```diff
@@ -36,15 +28,23 @@ export const createSignatureHeaders = async ({
   if (request.body) {
     components.push('content-digest', 'content-length', 'content-type')
   }
-  const { headers } = await httpsig.sign(request, {
-    components,
-    parameters: {
-      created: Math.floor(Date.now() / 1000)
+
+  const signingKey = createSigner(privateKey, 'ed25519', keyId)
+
+  const { headers } = await httpbis.signMessage(
+    {
+      key: signingKey,
+      name: 'sig1',
+      params: ['alg', 'keyid', 'created'],
```

Comment:
Interestingly:


> The explicit alg signature parameter MUST NOT be included in the signature, since the algorithm will be derived either from the key material or from the proof value.

[(source)](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-core-protocol-16#section-7.3.1)

Reply:
This seems logical, especially considering we provide the JWK during verification, which includes the algorithm.

All tests are passing without the `alg` parameter - check https://github.com/interledger/open-payments/pull/421/commits/81eceb29748ab7ece7c11251aacbd9d1a9f7dabc. Open Payments SDK tests are failing because it expects the `alg` parameter as well.